resources/md/guestbook.md

```diff
@@ -440,6 +440,10 @@ Now that our model is all set up, let's reload the application, and test our que
 ;; => [{:id 1, :name "m1", :message "hello world", :timestamp 1636480432353}]
 ```
 
+In this example, the newly defined `query-fn` function allows you to execute the SQL functions you defined in `queries.sql`. It achieves this using the `:db.sql/query-fn` method that comes with kit-sql (a dependency of kit/sqlite you installed). 
```

Comment:
Here I tried to explain the usage of query-fn in REPL before the reader moves on to "Exposing Database Queries in the Router Component", in an effort to make it easier to understand query-fn there. 

But I again feel like I don't understand that part :D in REPL, we define query-fn with `(def query-fn (:db.sql/query-fn state/system))`. In system.edn we are actually referencing the same component, but we pass on a different set of parameters (we customize :conn, and refer to a specific sql file). 

@yogthos is this correct?

Reply:
Yeah that looks correct. Basically, what's happening is that there's a bit of indirection. We define the `:db.sql/query-fn` to manage the queries, and then we reference it in the routes at `:query-fn`, and that's what ends up being exposed in the multimethod for the route. I tried clarifying this a bit more [here](https://github.com/kit-clj/kit-clj.github.io/blob/master/resources/md/integrant.md#accessing-components) as well.

Reply:
This is perfect, I'll add links to this clarification here and in the next section.

Reply:
👍